src/components/FocusBoundary.js

```diff
@@ -12,6 +13,9 @@ export const FocusBoundary = ({ children, onChange, onEnter, onExit }) => {
           onEnter && onEnter(e);
           boundaryContainsFocus.current = true;
         }
+        if (Object.values(e.path[0])[0].elementType === 'nav') {
+          backToTop();
+        }
```

Comment:
Couple of things here:
- Feels like this should not be included in the `FocusBoundary` component. Not sure where it would make sense to put it, but this is definitely not a component where this should be relevant.
- We should avoid anything which inspects the DOM wherever possible; getting the path and comparing the element type would make this likely to break in future.

Reply:
I moved the function out of FocusBoundary on my latest commit, but it still inspects the DOM, I couldn't figure out other way 😞.
HMU when someone else finds a better way, I'm curious 😃